bundle-core/src/main/java/net/discdd/utils/FileUtils.java

```diff
@@ -17,4 +31,56 @@ public static void createFileWithDefaultIfNeeded(Path path, byte[] bytes) throws
     public static void createEmptyFileIfNeeded(Path path) throws IOException {
         createFileWithDefaultIfNeeded(path, new byte[0]);
     }
+
+    public static String getFileNamesListFromDirectory(String directory) {
```

Comment:
We should be using FIle or preferably Path

Reply:
method deleted now